product_info.toml

```diff
@@ -30,7 +30,23 @@ id = "push-jobs-server"
 pretty_name = "Chef Push Jobs Server"
 filename = "push_jobs_server"
 
+[[products]]
+id = "chef-dk"
+pretty_name = "Chef Development Kit"
+filename = "chef_dk"
+mixlib_name = "chefdk"
+license_required_version = "4"
+
+[[products]]
+id = "chef-workstation"
+pretty_name = "Chef Workstation"
+filename = "chef_workstation"
+mixlib_name = "chef-workstation"
+license_required_version = "0.3"
```

Comment:
wasn't this going to be 0.4?

Reply:
Remind me again - what is going in `0.3`?

Reply:
remote policy file